content/en/docs/configuration/acme/_index.md

```diff
@@ -120,6 +128,27 @@ spec:
           class: nginx
 ```
 
+### Reusing an ACME Account
+
+You may want to reuse a single ACME account across multiple clusters. This
+might especially be useful when using EAB. If the `disableAccountKeyGeneration`
+field is set, cert-manager will not create a new ACME account and use the
+existing key specified in `privateKeySecretRef`.
```

Comment:
I think we should also note that the Issuer/ClusterIssuer will not be ready and retry until the Secret is provided.

Reply:
Great idea! Added text to mention that 👍